### ISSUE TITLE
Fix wrong documentation for the crypto module in safe_app

### ISSUE THREAD

**Issue by m-cat** [maidsafe/safe_client_libs#779]
* Some of the functions in the FFI crypto module have incorrect documentation. E.g. [`decrypt`](https://docs.rs/safe_app/0.6.0/safe_app/ffi/crypto/fn.decrypt.html) says that a user should provide a recipient's secret key which is wrong.
* See if documentation for other functions should be updated too.

**Comment by m-cat** [comment]
- The `decrypt` documentation actually seems to be correct. Here is the example usage code provided in rust_sodium for reference:

```rust
use rust_sodium::crypto::box_;
let (ourpk, oursk) = box_::gen_keypair();
// normally theirpk is sent by the other party
let (theirpk, theirsk) = box_::gen_keypair();
let nonce = box_::gen_nonce();
let plaintext = b"some data";
let ciphertext = box_::seal(plaintext, &nonce, &theirpk, &oursk);
let their_plaintext = box_::open(&ciphertext, &nonce, &ourpk, &theirsk).unwrap();
assert!(plaintext == &their_plaintext[..]);
```

- The other documentation seems to be correct, but I will fix up the style a bit.

**Comment by nbaksalyar** [comment]
Closed via #978
